docker-compose.override.test.yml

```diff
@@ -15,3 +15,7 @@ services:
     build:
       context: .
       dockerfile: docker/nginx.docker
+
+  celery:
+    profiles:
```

Comment:
don't you also need to add rabbitmq and redis to the `donotstart` profile?

Reply:
I tried in [this](https://github.com/multilateralfund/ors/pull/228/commits/d5890d72d42956be19bd9d79fab6186765cc8699) commit, but the tests will fail.